README.md

```diff
@@ -137,6 +138,38 @@ var text: UITextField
 
 ------
 
+### amb
+
+Amb takes multiple publishers and mirrors the first one to emit an event. You can think of it as a race of publishers, where the first one to emit passes its events, while the others are ignored.
```

Comment:
Might be worth a small mention on `amb`’s etymology—for newcomers, it’s a bit of an odd term.